test/cypress/integration/field-extension.spec.js

```diff
@@ -24,4 +24,23 @@ context('Field extension', () => {
       .find('[data-test-id="cf-ui-text-input"]')
       .should('exist')
   })
+
+  it('opens a page extension and tests navigating within the page', () => {
+    cy.visit(pageExtension('test-page-extension'))
+
+    // eslint-disable-next-line
+    cy.wait(3000)
+
+    cy.get('[data-test-id="page-extension"] iframe').captureIFrameAs('extension')
+
+    cy.get('@extension')
+      .find('[data-test-id="cf-ui-page-extension"]')
+      .should('exist')
+
+    cy.get('@extension')
+      .find('button:first')
+      .click()
```

Comment:
would be cool to verify that `path` is passed to `invocation` params. we could check the output of `JSON.stringify` output

Reply:
Yeah I think that's a good idea, I can add that in.

Reply:
On second thought, I will skip this for now. We should have tests for the method.